test/messaging/components/MoveToOption.unit.spec.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import MoveToOption from '../../../src/js/messaging/components/MoveToOption.jsx';
+
+const props = {
+  folderName: 'Folder',
+  folderId: 1,
+};
+
+describe('<MoveToOption>', () => {
+  it('should render correctly', () => {
+    const tree = SkinDeep.shallowRender(
+      <MoveToOption {...props}/>
+    );
+
+    expect(tree.getRenderOutput()).to.exist;
+  });
+
+  it('should render the expected input field', () => {
+    const tree = SkinDeep.shallowRender(
+      <MoveToOption {...props}/>
+    );
+    expect(tree.subTree('input[name=messagingMoveToFolder]')).to.exist;
```

Comment:
I think the success cases for `subTree` calls should be tested as `expect(tree.subTree(...)).to.be.ok`, since failure results in `false`, but `exist` only checks for `undefined` and `null`.